src/solvers/base_solver.cpp

```diff
@@ -127,25 +115,20 @@ int BaseSolver::GetCycle() const
   return m_cycle;
 }
 
-void BaseSolver::InitializeOutput(const OutputType output_type, std::string root_name,
-                                  const mfem::Array<std::string> names)
+void BaseSolver::InitializeOutput(const OutputType output_type, std::string root_name)
```

Comment:
Did names disappear as a conflict or should the assert be removed?

Reply:
more leftover std::map stuff